int32 frame_id
string src_address
int32 comm_bytes